### PR TITLE
Improve Rollbar error logging support

### PR DESCRIPTION
**Pull request by maurizi**
- Get dependency from jCenter instead of a checked in JAR file
 - Include LogCat logs for API versions higher than 15
 - Include `instance_url_name` and `username` if available when reporting
   errors

Connects to #260

### CODE REVIEW

**Comment by maurizi** [comment]
@azavea-bot rebuild

File: OpenTreeMap/src/main/java/org/azavea/helpers/Logger.java [hunk 1]
```diff
@@ -61,9 +67,28 @@ public static void error(String message, Throwable exception) {
...
+        String urlName = null;
+        String username = null;
+        if (App.getCurrentInstance() != null) {
+            urlName = App.getCurrentInstance().getUrlName();
+        }
+        if (App.getLoginManager() != null && App.getLoginManager().loggedInUser != null) {
+            try {
+                username = App.getLoginManager().loggedInUser.getUserName();
+            } catch (JSONException e) {
+                // This should happen, but if it does we'll just let username be null
```
Comment (maurizi) [inline OpenTreeMap/src/main/java/org/azavea/helpers/Logger.java]: should -> shouldn't

Comment (flibbertigibbet) [inline OpenTreeMap/src/main/java/org/azavea/helpers/Logger.java]: Might want to fix this comment, and the one above (`url_name` -> `urlName`)

**Comment by flibbertigibbet** [comment]
+1

Just curious: why jCenter instead of maven central?

**Comment by maurizi** [comment]
@flibbertigibbet Rollbar only released their library on jCenter. Since jCenter _also_ mirrors Maven Central, it's a pretty easy switch.

FYI I think new Android projects default to jCenter now when you create them from Android Studio.

**Pull request merged**